### PR TITLE
re-enable slave restart test

### PR DESCRIPTION
**Pull request by bparees**

### CODE REVIEW

File: test/extended/image_ecosystem/mysql_replica.go [hunk 1]
```diff
@@ -13,7 +13,7 @@ import (
 	testutil "github.com/openshift/origin/test/util"
 
 	kapiv1 "k8s.io/api/core/v1"
-	//metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```
Comment (smarterclayton) [inline test/extended/image_ecosystem/mysql_replica.go]: ?

Comment (bparees) [reply test/extended/image_ecosystem/mysql_replica.go]: the code that was commented out (and is being renabled in this PR) uses this package, so the package import is being un-commented out.

**Comment by bparees** [comment]
@smarterclayton from the failing test's registry logs:

```
Jun 22 17:31:09.175: INFO: Running 'oc logs --config=/tmp/cluster-admin.kubeconfig --namespace=default dc/docker-registry --since=1m2.175299876s'
```

registry running for 1 minute?  Seems likely the registry got restarted..   are we perhaps running out of resources somewhere and causing pod evictions?  That could explain a lot of the flakiness we've been seeing lately.

/retest

**Comment by bparees** [comment]
/retest

Ben Parees | OpenShift

**Pull request closed without merging**